src/Umbraco.Web.UI.Client/src/common/services/minieditorhelper.service.js

```diff
@@ -0,0 +1,79 @@
+(function () {
+    'use strict';
+
+    function miniEditorHelper(dialogService, editorState, fileManager, contentEditingHelper) {
+
+        var launched = false;
+
+        function launchMiniEditor(node) {
```

Comment:
I know this is a copy/paste job into this new helper service, but again is it worth documenting loosely for our API auto-gen'd docs @madsrasmussen ?